pkg/arvo/tests/lib/naive.hoon

```diff
@@ -3692,13 +3693,19 @@
         [[c b %marbud-key-0] +(c)]
       [[c b %marbud-key-1] +(c)]
   ::
+  =/  txs
+    (cad:naive 3 (gen-tx 0 tx-1 %marbud-key-0) (gen-tx 1 tx-2 %marbud-key-1) ~)
+  =/  bat=octs  (cad:naive 3 (reap (add 1 (div batch-size 2)) txs))
```

Comment:
This gives the wrong nonce after the first two transactions. I suppose it doesn't matter if all we're testing is parsing speed since parsing happens before nonce is checked, but it would also be good to know that there are no issues when a batch consists of a large number of successful transactions too.

Reply:
Yeah.  The issue right now is that processing transactions takes much longer than parsing, because verifying signatures requires two (unjetted) keccak calls (one before because we sign a hash and one after because we hash to turn the recovered point into an ethereum address), so the test takes too long to run at the batch sizes we need to test (5k is probably sufficient).

Now that I realize keccak is used at runtime, it's a priority to jet it, so we'll get that done soon.  When that happens, I expect we'll be able to do your full version with 5k txs and it'll take around 15 seconds, which is worth doing.  The rest of the test suite will be a lot faster too, because it spends virtually all its time in keccak.

Until then, we should make sure this test (1) passes, (2) is ready to be used in its full form to make sure it processes all txs (like you had it), and (3) is ready to be used to test only parsing speed.  For (3), generating valid txs is time-consuming and unnecessary, and in fact we should probably just call parse-roll directly.  Probably just having two separate tests is good.

Reply:
I split the test into two and tweaked the output.